### PR TITLE
update install instructions for golang newbies

### PR DESCRIPTION
**Pull request by robert-gurol-signavio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,16 +5,35 @@ Those credentials will be valid for 12 hours by default.
...
+
+Make sure your go path is part of your PATH environment variable: 
+```
+export PATH="${PATH}:~/go/bin/"
+```
+
+### .. or download from releases
+
+```console
 curl -L https://github.com/signavio/aws-mfa-login/releases/latest/download/aws-mfa-login_linux_amd64.gz -o aws-mfa-login.gz
```
Comment (karlderkaefer) [inline README.md]: Could you make me favor and replace Linux with $(uname) with will work for Mac and Linux

File: README.md [hunk 2]
```diff
@@ -5,16 +5,35 @@ Those credentials will be valid for 12 hours by default.
...
-or download from releases
-```yaml
+.. or update by setting the -u flag: 
+```bash
+go get -u github.com/signavio/aws-mfa-login
+``` 
+
+Make sure your go path is part of your PATH environment variable: 
+```
+export PATH="${PATH}:~/go/bin/"
```
Comment (karlderkaefer) [inline README.md]: add also export GOPATH=~/go because this might depend on which install method you used.

Comment (robert-gurol-signavio) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (robert-gurol-signavio) [reply README.md]: ok, done

**Comment by karlderkaefer** [comment]
Pls merge

**Pull request merged**